lua/astrocommunity/colorscheme/hybrid-nvim/init.lua

```diff
@@ -0,0 +1,3 @@
+return {
+  "HoNamDuong/hybrid.nvim",
```

Comment:
```suggestion
  "HoNamDuong/hybrid.nvim",
  opts = {},
```

this should be called